src/platform/silabs/EFR32/BUILD.gn

```diff
@@ -32,29 +32,29 @@ if (chip_crypto == "platform") {
 
 static_library("EFR32") {
   sources = [
-    "../FreeRTOS/SystemTimeSupport.cpp",
-    "../SingletonConfigurationManager.cpp",
+    "../../FreeRTOS/SystemTimeSupport.cpp",
```

Comment:
I suggest making a variable for the path instead of `../`

Reply:
Done on the latest commit